src/pages/admin/CreateNewMeetingPage.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import apis, { CreateMeeting, Meeting } from '@/lib/apis'
+import BaseInput from '@/components/UI/BaseInput.vue'
+import BaseButton from '@/components/UI/BaseButton.vue'
+import EmoineHeader from '@/components/EmoineHeader.vue'
+import router from '@/router'
+
+//todo: 専用ヘッダー画像のアップロード
+const headerLogo = '/logo.png'
+
+const liveUrl = ref('')
+const isUrlValidated = ref(true)
+
+function getLiveIdFromUrl(): string {
```

Comment:
今回だとどっちでもいいんだけど個人的にはアロー関数の方が好きってことだけ書いとく
```suggestion
const getLiveIdFromUrl = (): string => {
```